common/src/main/java/org/opensearch/ml/common/parameter/MLModelOutput.java

```diff
@@ -0,0 +1,54 @@
+package org.opensearch.ml.common.parameter;
+
+import lombok.Builder;
+import lombok.Getter;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.common.xcontent.XContentBuilder;
+
+import java.io.IOException;
+
+@Getter
+public class MLModelOutput extends MLOutput {
```

Comment:
We only need to add new `MLOutput` for new algorithm. For this get model API, we don't introduce new algorithm. So no need to add this class.

Reply:
Correct. I think we still need to include MLModel in the MLModelGetResponse. Based on the current structure, the Response is defined in the ml-common lib, which is a dependency to ml-plugin where the MLModel is defined. So MLModelGetResponse cannot find the MLMode. My thinking is that we can move MLModel into ml-common, so it needs some refactor.

Reply:
Sure, feel free to move `MLModel` to `commons` package.